items/src/test/scala/weco/api/items/ItemsApiFeatureTest.scala

```diff
@@ -39,39 +43,72 @@ class ItemsApiFeatureTest
     with WorkGenerators
     with ItemsGenerators {
 
+  private def createPhysicalItemWith(
+    sierraItemIdentifier: String,
+    accessCondition: AccessCondition
+  ) = {
+    val physicalItemLocation: PhysicalLocation = createPhysicalLocationWith(
+      accessConditions = List(accessCondition),
+      locationType = LocationType.ClosedStores
+    ).copy(license = None, shelfmark = None)
+
+    // Sierra identifiers without check digits are always 7 digits
+    require(sierraItemIdentifier.matches("^\\d{7}$"))
+
+    val physicalItemSierraIdentifierWithCheckDigit = f"${sierraItemIdentifier}5"
+    val physicalItemSierraSourceIdentifier =
+      f"i${physicalItemSierraIdentifierWithCheckDigit}"
```

Comment:
Could you use the SierraItemNumber types here instead?

Reply:
Yes we should.